src/Core/Injector/InjectorNotFoundException.php

```diff
@@ -3,6 +3,7 @@
 namespace SilverStripe\Core\Injector;
 
 use Psr\Container\NotFoundExceptionInterface;
+use Throwable;
```

Comment:
What is this addition needed for?

Reply:
Ah that's a hangover from some code that was added and subsequently removed, I think.